lib/src/rive_core/state_machine_controller.dart

```diff
@@ -256,6 +257,8 @@ class StateMachineController extends RiveAnimationController<CoreContext> {
 
   late CoreContext core;
 
+  final _recognizer = ImmediateMultiDragGestureRecognizer();
```

Comment:
Is it ok for this to be stored/long lived?

Reply:
@luigi-rosso I assume yes, since the pointer down events are handled independently (In `GestureDetector` they are also assigned on `initState`). We could instantiate the recognizer only on the first `PointerDownEvent` and only if there are `HitShapes`. But not sure if we gain anything from that...
And I forgot to dispose it, so this is fixed now!